platform/mellanox/mlnx-platform-api/sonic_platform/sfp.py

```diff
@@ -620,6 +621,18 @@ def _set_lpmode_raw(cls, sdk_handle, sdk_index, slot_id, ports, attr_type, power
 
         return result
 
+    def check_lpmode(self, desired_lpmode, api):
+        return api.get_lpmode() == desired_lpmode
+
+
+    def wait_until(self, timeout, poll_interval, desired_lpmode, api):
```

Comment:
There are already 2 wait_util implementations, please re-use the current implementation instead of adding a new function.  
https://github.com/sonic-net/sonic-buildimage/blob/master/platform/mellanox/mlnx-platform-api/sonic_platform/utils.py#L274
https://github.com/sonic-net/sonic-buildimage/blob/master/platform/mellanox/mlnx-platform-api/sonic_platform/utils.py#L293

Reply:
Done